config.h

```diff
@@ -0,0 +1,8 @@
+#pragma once
+#include<iostream>
+class Config {
+public:
+	std::string tasksFileName = "tasks.txt";
```

Comment:
Może ten config do maszyny stanów przekaż bo teraz ta klasa nie jest zbyt przydatna.
W konstruktorze przekaż nazwy tych plików.
Albo jak chcesz to możesz też zrobić te pola statyczne i wcale nie tworzyć obiektu.
Do wyboru